### PR TITLE
Certification fixes

### PR DESCRIPTION
**Pull request by nCastle1**

### CODE REVIEW

File: src/Forms/Shared/Samples/Map/AuthorMap/AuthorMap.xaml.cs [hunk 1]
```diff
@@ -328,23 +330,27 @@ private void AddLayer(string layerName, string url)
...
             // Create a new map image layer
-            layer = new ArcGISMapImageLayer(layerUri);
-            layer.Name = layerName;
+            layer = new ArcGISMapImageLayer(layerUri)
+            {
+                Name = layerName,
 
-            // Set it 50% opaque, and add it to the map
-            layer.Opacity = 0.5;
+                // Set it 50% opaque, and add it to the map
```
Comment (fragmer) [inline src/Forms/Shared/Samples/Map/AuthorMap/AuthorMap.xaml.cs]: Comment got detached from "adding it to the map". I'd avoid object initializer here. What are we gaining from it?

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Geometry/BufferList/BufferList.xaml.cs [hunk 2]
```diff
@@ -106,7 +108,7 @@ public BufferList()
             try
             {
                 // Get the boolean value whether to create a single unioned buffer (true) or independent buffer around each map point (false).
-                bool unionBufferBool = (UnionCheckBox.IsChecked == true);
+                bool unionBufferBool = UnionCheckBox.IsChecked == true;
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Geometry/BufferList/BufferList.xaml.cs]: Keep parens.

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs [hunk 3]
```diff
@@ -139,7 +139,7 @@ private void GetSuitableTransformations(SpatialReference inSpatialRef, SpatialRe
             {
                 DatumTransformationListBoxItem item = new DatumTransformationListBoxItem(transform)
                 {
-                    IsDefault = (transform.Name == defaultTransform.Name)
+                    IsDefault = transform.Name == defaultTransform.Name
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs]: Keep parens.

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WMSLayerUrl/WMSLayerUrl.xaml.cs [hunk 4]
```diff
@@ -37,11 +37,13 @@ public WMSLayerUrl()
...
-            Map myMap = new Map(Basemap.CreateImagery());
+            Map myMap = new Map(Basemap.CreateImagery())
+            {
 
-            // Set the initial viewpoint
-            myMap.InitialViewpoint = new Viewpoint(
-                new MapPoint(25.450, -4.59, new SpatialReference(4326)), 1000000);
+                // Set the initial viewpoint
+                InitialViewpoint = new Viewpoint(
+                new MapPoint(25.450, -4.59, new SpatialReference(4326)), 1000000)
```
Comment (fragmer) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WMSLayerUrl/WMSLayerUrl.xaml.cs]: Fix indentation and use `SpatialReferences.Wgs84`

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs [hunk 5]
```diff
@@ -142,7 +142,7 @@ public ListTransformations()
             {
                 DatumTransformationListBoxItem item = new DatumTransformationListBoxItem(transform)
                 {
-                    IsDefault = (transform.Name == defaultTransform.Name)
+                    IsDefault = transform.Name == defaultTransform.Name
```
Comment (fragmer) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs]: Keep parens

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/WMSLayerUrl/WMSLayerUrl.xaml.cs [hunk 6]
```diff
@@ -38,11 +38,13 @@ public WMSLayerUrl()
...
-            Map myMap = new Map(Basemap.CreateImagery());
+            Map myMap = new Map(Basemap.CreateImagery())
+            {
 
-            // Set the initial viewpoint
-            myMap.InitialViewpoint = new Viewpoint(
-                new MapPoint(25.450, -4.59, new SpatialReference(4326)), 1000000);
+                // Set the initial viewpoint
+                InitialViewpoint = new Viewpoint(
+                new MapPoint(25.450, -4.59, new SpatialReference(4326)), 1000000)
```
Comment (fragmer) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/WMSLayerUrl/WMSLayerUrl.xaml.cs]: Fix indentation and use `SpatialReferences.Wgs84`

File: src/iOS/Xamarin.iOS/Samples/Security/TokenSecuredChallenge/TokenSecuredChallenge.cs [hunk 7]
```diff
@@ -300,20 +300,22 @@ public LoginOverlay(CGRect frame, nfloat transparency, UIColor color, string url
             nfloat buttonWidth = 60;
 
             // Get the total height and width of the control set (five rows of controls, four sets of space).
-            nfloat totalHeight = (5 * controlHeight) + (4 * rowSpace);
+            nfloat totalHeight = 5 * controlHeight + 4 * rowSpace;
```
Comment (fragmer) [inline src/iOS/Xamarin.iOS/Samples/Security/TokenSecuredChallenge/TokenSecuredChallenge.cs]: IMO this was easier to read with parens. What is gained by removing them? Why make the reader check operator precedence in their head?

File: src/Forms/Shared/Samples/Data/EditAndSyncFeatures/EditAndSyncFeatures.xaml.cs [hunk 8]
```diff
@@ -128,7 +130,7 @@ public EditAndSyncFeatures()
             }
             catch (Exception ex)
             {
-                await DisplayAlert("Error", ex.ToString(), "OK");
+                await ((Page)Parent).DisplayAlert("Error", ex.ToString(), "OK");
```
[the remark this answers is not in the document]
Comment (nCastle1) [reply src/Forms/Shared/Samples/Data/EditAndSyncFeatures/EditAndSyncFeatures.xaml.cs]: Good catch, I used find/replace but only caught the awaited calls.

File: src/Forms/Shared/Samples/Map/AuthorMap/AuthorMap.xaml [hunk 9]
```diff
@@ -6,15 +6,18 @@
              xmlns:mapping="clr-namespace:Esri.ArcGISRuntime.Mapping;assembly=Esri.ArcGISRuntime">
   <Grid>
     <Grid.RowDefinitions>
-      <RowDefinition/>
-      <RowDefinition Height="70"/>
+      <RowDefinition Height="*"/>
```
[the remark this answers is not in the document]
Comment (nCastle1) [reply src/Forms/Shared/Samples/Map/AuthorMap/AuthorMap.xaml]: Good catch. This appears to be a Xamarin.Forms bug. https://github.com/xamarin/Xamarin.Forms/issues/2624

I'll try upgrading the package version to work around this.

**Pull request merged**